packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -460,15 +466,18 @@ function returnModalCopy(
 	productType: string,
 	translate: ReturnType< typeof useTranslate >,
 	createUserAndSiteBeforeTransaction: boolean,
-	isRenewal = false
+	isRenewal = false,
+	product?: ResponseCartProduct
 ): ModalCopy {
+	const domainNameString = product ? product.meta : 'your selected domain';
```

Comment:
I think this string should probably be translated.

Reply:
Ah great catch, I'll update that.